src/Components/NavBar/Navbar.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Navbar.css";
+
+class Navbar extends React.Component {
+  render() {
+    return (
+      <div className="navbar navbar-expand-lg navbar-light bg-light">
+        <h3>GPA Calculator</h3>
+      </div>
```

Comment:
Hmmmm I am a bit on the fence, if this really needs an own component, as there is only an h3 being displayed 😅